x/programs/rust/examples/nft/Cargo.toml

```diff
@@ -17,5 +17,5 @@ serial_test = "2.0.0"
 crate-type = ["cdylib"] # set the crate(needed for cargo build to work properly)
 
 [[example]]
```

Comment:
since we aligned on using integration tests we can remove this and the examples dir and replace with [[integration]]